lib/pengine/tests/native/native_find_rsc_test.c

```diff
@@ -509,6 +495,13 @@ bundle_replica_rsc(void **state) {
     assert_ptr_equal(child_0, native_find_rsc(httpd_bundle, "httpd:0", httpd_bundle_0, pe_find_current));
     assert_ptr_equal(container_0, native_find_rsc(httpd_bundle, "httpd-bundle-docker-0", cluster01, pe_find_current));
     assert_ptr_equal(remote_0, native_find_rsc(httpd_bundle, "httpd-bundle-0", cluster01, pe_find_current));
+    return false; // Do not iterate through any further replicas
+}
+
+static void
+bundle_replica_rsc(void **state)
+{
+    pe__foreach_bundle_replica(httpd_bundle, bundle_first_replica, NULL);
```

Comment:
The intent would be a bit clearer at this level if we had a `pe__first_replica()` (like `pe__first_container()`). Since this would be the only caller and it's a unit test, I'm not concerned about it.